Date.php

```diff
@@ -1274,7 +1273,6 @@ function truncSeconds($pn_precision = 0,
      */
     function getDate($format = DATE_FORMAT_ISO)
```

Comment:
This method should add the `public` or `private` to let developers know the method property.